pkgs/applications/networking/lls/default.nix

```diff
@@ -0,0 +1,22 @@
+{ rustPlatform, fetchFromGitHub, lib }:
+rustPlatform.buildRustPackage {
+  pname = "lls";
+  version = "0.1.0";
+
+  src = fetchFromGitHub {
+    owner = "jcaesar";
+    repo = "lls";
+    rev = "badb937f8c35c2dccc2ae259608cfb606a837f3e";
+    hash = "sha256-Aq0MGhzSoJCkM0Wt/r5JSOz96LyRSgSryD7+m4aFZEA=";
```

Comment:
Let's see if we can get tags.

https://github.com/jcaesar/lls/issues/1

Reply:
looks like we can use `fetchCrate` here: https://crates.io/crates/lls